pyproject.toml

```diff
@@ -36,12 +36,11 @@ Releases = "https://github.com/smroid/cedar-solve/releases"
 cedar-detect = [
     "grpcio",
     "protobuf",
-    "grpcio-tools",
 ]
 dev = [
-
     "pytest",
     "build",
+    "grpcio-tools",
```

Comment:
i moved this into `dev` so that the user will not install this when installing the wheel as they don't need to rebuild the grpc generated code when using this as a library.